src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,788 @@
+        <td>
+            <p>Enables the plugin to read and write to the clipboard.</p>
+            <p>Possible values:</p>
+            <ul>
+                <li><inlineCode>read</inlineCode>: enables the plugin to read from the clipboard.</li>
+                <li><inlineCode>readAndWrite</inlineCode>: enables the plugin to read from and write to the clipboard.</li>
+            </ul>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (no clipboard access)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>localFileSystem</inlineCode></td>
+        <td><inlineCode>"plugin" | "request" | "fullAccess"</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Enables the plugin to access the file system.</p>
+            <p>Possible values:</p>
+            <ul>
+                <li><inlineCode>plugin</inlineCode>: enables the plugin to access the file system in the plugin folder.</li>
+                <li><inlineCode>request</inlineCode>: enables the plugin to request access to the file system (user consent).</li>
+                <li><inlineCode>fullAccess</inlineCode>: enables the plugin to access the file system without requesting access.</li>
+            </ul>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>"plugin"</inlineCode></p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>network</inlineCode></td>
+        <td><inlineCode>NetworkPermission</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Enables the plugin to access the network.</p>
+            <p>For example, to make HTTP requests (XHR, fetch, etc.), load images (<inlineCode>&lt;img src="" /&gt;</inlineCode>), etc.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (no network access)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>webview</inlineCode></td>
+        <td><inlineCode>WebviewPermission</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Enables the plugin to use webviews in its UI to display web content or complex UI.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (no webview usage)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>launchProcess</inlineCode></td>
+        <td><inlineCode>LaunchProcessPermission</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Enables the plugin to launch processes using APIs like <inlineCode>require("uxp").shell.openPath()</inlineCode> or <inlineCode>shell.openExternal()</inlineCode>.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (no process launching)</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### NetworkPermission
+
+<p>Specifies the domains that the plugin can access in network requests.</p>
```

Comment:
These are some other examples of this property that we should include
"network": {
            "domains": [
                "https://www.adobe.com/",
                "https://*.adobeprerelease.com",
                "wss://*.myplugin.com"
            ]
        }

Reply:
My comment deleted the asterisks :(

Reply:
No worries, I assumed as much and added them back.

I know this problem well because the asterisk is a very important "ingredient" in gender-neutral language in German, meaning I'm more than familiar with interpreting "mis-interpreted asterisks" 😉